### ISSUE TITLE
Sending tablist remove packet for players with 1.19.3 listed=false 

### ISSUE THREAD

**Issue by fullwall** [ViaVersion/ViaBackwards#510]
### Problem Description

A tablist remove packet is not sent when a player is added with tablist=false. From a semantics point of view I feel that removing them from the player list with a packet for older versions would be better. Currently I have to implement this myself.

### Solution Description

Send a Tablist remove packet when listed=false on 1.19.3

### Alternatives

I have implemented this myself currently by checking the protocol version of the player.

### Additional Info

Minecraft 1.19.3

**Comment by MenschenToaster** [comment]
I agree. What should be noted is, that the removal packet should be delayed for a few seconds to achieve more or less the same effect

**Comment by SplotyCode** [comment]
Duplicate of https://github.com/ViaVersion/ViaBackwards/issues/510

**Comment by MenschenToaster** [comment]
> Duplicate of #510

This is Issue Nr 510 my guy
